orchestrator-swarm/src/main/java/com/sequenceiq/cloudbreak/orchestrator/swarm/SwarmContainerOrchestrator.java

```diff
@@ -173,18 +174,33 @@ public void startConsulWatches(ContainerOrchestratorCluster cluster, String imag
     public boolean areAllNodesAvailable(String gatewayAddress, Set<Node> nodes) {
         LOGGER.info("Checking if Swarm manager is available and if the agents are registered.");
         try {
+            List<String> allAvailableNode = getAllAvailableNode(gatewayAddress, nodes);
+            if (allAvailableNode.size() == getPrivateAddresses(nodes).size()) {
+                return true;
+            }
+        } catch (Throwable t) {
+            LOGGER.info(String.format("Exception occurred under the swarm-manager request: %s", t.getMessage()));
```

Comment:
log at least a warning and change the message to: Cannot connect to Swarm manager: %s